apiv2/internal/server/customconfig.go

```diff
@@ -6,41 +6,149 @@ package server
 import (
 	"context"
 
-	"google.golang.org/grpc/codes"
-
-	customconfig_v1 "github.com/open-edge-platform/infra-core/apiv2/v2/internal/pbapi/resources/customconfig/v1"
+	customconfigv1 "github.com/open-edge-platform/infra-core/apiv2/v2/internal/pbapi/resources/customconfig/v1"
 	restv1 "github.com/open-edge-platform/infra-core/apiv2/v2/internal/pbapi/services/v1"
+	inv_computev1 "github.com/open-edge-platform/infra-core/inventory/v2/pkg/api/compute/v1"
+	inventory "github.com/open-edge-platform/infra-core/inventory/v2/pkg/api/inventory/v1"
 	"github.com/open-edge-platform/infra-core/inventory/v2/pkg/errors"
+	"github.com/open-edge-platform/infra-core/inventory/v2/pkg/validator"
 )
 
+func toInvCustomConfig(customConfig *customconfigv1.CustomConfigResource) (*inv_computev1.CustomConfigResource, error) {
+	if customConfig == nil {
+		return &inv_computev1.CustomConfigResource{}, nil
```

Comment:
If the received custom configuration is a nil pointer, should we not return an error stating this?